src/tests/user.test.js

```diff
@@ -26,6 +26,101 @@ describe('/POST Signup route', () => {
       });
   });
 
+  it('should return an error first name is missing', done => {
```

Comment:
Line 29 to 123 should be just one test case.
```suggestion
  it('should return an error if credentials are invalid', done => {
```

Reply:
Correction received, thanks